i6_models/assemblies/conformer/conformer.py

```diff
@@ -0,0 +1,110 @@
+from __future__ import annotations
+
+__all__ = ["ConformerBlockV1Config", "ConformerEncoderV1Config", "ConformerBlockV1", "ConformerEncoderV1"]
+
+import torch
+from torch import nn
+from dataclasses import dataclass
+
+from i6_models.config import ModelConfiguration
+from i6_models.parts.conformer import (
+    ConformerConvolutionV1,
+    ConformerConvolutionV1Config,
+    ConformerFrontendV1,
+    ConformerFrontendV1Config,
+    ConformerMHSAV1,
+    ConformerMHSAV1Config,
+    ConformerPositionwiseFeedForwardV1,
+    ConformerPositionwiseFeedForwardV1Config,
+)
+
+
+@dataclass
+class ConformerBlockV1Config(ModelConfiguration):
+    """
+    :param ff_cfg: Configuration for ConformerPositionwiseFeedForwardV1
+    :param mhsa_cfg: Configuration for ConformerMHSAV1
+    :param conv_cfg: Configuration for ConformerConvolutionV1
+    """
+
+    # nested configurations
+    ff_cfg: ConformerPositionwiseFeedForwardV1Config
+    mhsa_cfg: ConformerMHSAV1Config
+    conv_cfg: ConformerConvolutionV1Config
+
+
+@dataclass
+class ConformerEncoderV1Config(ModelConfiguration):
```

Comment:
Do we have a convention on whether configs should be at the top or right before the corresponding module? I would prefer the latter.